src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,19 @@
+export default class PokemonClient {
+  constructor() {}
+
+  async pokemonFetcherById(pokemon) {
```

Comment:
is pokemon can be multiple ids? rename if so it will be well understood.
worth create 1 main function to fetch 1 pokemon and 1 method to use when there are multiple pokmons (which will use the first method)